### ISSUE TITLE
blueprintjs rendered keyboard navigation is broken

### ISSUE THREAD

**Issue by dlech** [lukasbach/react-complex-tree#38]
**Describe the bug**
Keyboard navigation doesn't work as expected when using the blueprintjs renderer.

When the tree gets focus by tabbing with the keyboard, pressing down and up just scrolls the page instead of navigating the tree.

If you select a node with the mouse first, then navigation sort of works, but it seems to get stuck on expandable nodes.

**To Reproduce**

Go to https://rct.lukasbach.com/docs/guides/blueprintjs and try to use keyboard navigation.

**Expected behavior**

The keyboard navigation should work as in other examples.

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Browser Firefox
- Version 98

**Comment by dlech** [comment]
I also tried adding the renderers as suggested in the docs to a sandbox and it crashes:

https://codesandbox.io/s/react-complex-tree-playground-forked-cs3i6k?file=/src/App.tsx

> ### Error
>
> Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
>
> Check the render method of `TreeItem`.

**Comment by lukasbach** [comment]
I believe the blueprintjs related issue was fixed by your PR, and the incorrect keyboard navigation jumps should be fixed as well through another fix, can you please verify?
